app/src/main/java/com/example/chrono/main/timer/CircuitCreate.kt

```diff
@@ -16,6 +16,8 @@ class CircuitCreate : BaseActivity() {
 
     private var bind: ActivityCircuitCreateBinding? = null
 
+    private val timeChangeVal: Int = 5
```

Comment:
Is this such that we can change this value in the future if we want?

Reply:
Yeah